content/TruenasApps/StableApps/ElasticSearch.md

```diff
@@ -0,0 +1,181 @@
+The TrueNAS Elastic Search app allows you to configure and deploy a single Elasticsearch node.
+You can install multiple instances to deploy additional nodes, however you must configure a [Custom App]({{< relref "/TruenasApps/UsingCustomApp.md" >}}) with the **Install via YAML** option to deploy a [multi-node cluster](https://www.elastic.co/guide/en/elasticsearch/reference/master/docker.html#docker-compose-file).
+
+This tutorial covers installing the TrueNAS Elastic Search app to deploy a node.
+It does not detail management of the node or integrating it with other containers.
+Elastic provides a basic primer, [What is Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/master/elasticsearch-intro-what-is-es.html), with further information about the app and its place within the Elastic Stack.
+
+## Before You Begin
+
+Prepare TrueNAS before installing the app by:
+
+{{< include file="/static/includes/apps/BeforeYouBeginStableApps.md" >}}
+{{< include file="/static/includes/apps/BeforeYouBeginRunAsUser.md" >}}
+
+<div style="margin-left: 33px">{{< trueimage src="/images/SCALE/Apps/ElasticSearchDetailsScreen.png" alt="Elastic Search App Details Screen" id="Elastic Search App Details Screen" >}}</div>
+
+{{< include file="/static/includes/apps/BeforeYouBeginAddAppDatasets.md" >}}
+
+  <p style="margin-left: 33px">Create a storage dataset for Elastic Search Data Storage with a name such as <b><i>data</i></b>).
+  Select <b>apps</b> as the <b>Dataset Preset</b> for this dataset.
+  You can modify the dataset ACLs at the time of creation, or modify them later when adding them in the app.</p>
+
+<div style="margin-left: 33px">{{< include file="/static/includes/apps/BeforeYouBeginAddAppDatasetsProcedure.md" >}}</div>
+
+{{< include file="/static/includes/apps/BeforeYouBeginAddAppCertificate.md" >}}
+
+  <p style="margin-left: 33px">Adding a certificate is optional but if you want to use a certificate for this application, either create a new self-signed CA and certificate or import an existing CA and create the certificate for Elastic Search. A certificate is not required to deploy the application.</p>
+
+## Installing the Application
+
+{{< hint info >}}
+This basic procedure covers the required Elastic Search app settings.
+For optional settings, see [Understanding App Installation Wizard Settings](#understanding-app-installation-wizard-settings).
+{{< /hint >}}
+
+{{< include file="static/includes/apps/MultipleAppInstancesAndNaming.md" >}}
+
+{{< include file="/static/includes/apps/LocateAndOpenInstallWizard.md" >}}
+
+{{< trueimage src="/images/SCALE/Apps/InstallElasticSearchScreen.png" alt="Install Elastic Search Screen" id="Install Elastic Search Screen" >}}
+
+{{< include file="/static/includes/apps/InstallWizardAppNameAndVersion.md" >}}
+
+Enter the **Elastic Search Configuration** settings.
+
+Enter a password to use for the built-in `elastic` user.
```

Comment:
why are you using the code style for the user? Is it entered in a command string? If not, use bold.